src/CalcViewModel/DataLoaders/UnitConverterDataLoader.cpp

```diff
@@ -156,27 +156,32 @@ void UnitConverterDataLoader::GetCategories(_In_ shared_ptr<vector<UCM::Category
 
 void UnitConverterDataLoader::GetUnits(_In_ unordered_map<ViewMode, vector<OrderedUnit>>& unitMap)
 {
-    bool USSource, USTarget;
-    bool UKSource, UKTarget;
-    bool Source, Target;
+    // US + Federated States of Micronesia, Marshall Islands, Palau
+    bool isPartOfCOFA = m_currentRegionCode == L"US" ||
```

Comment:
I don't think you need the isPartOfCOFA variable--you can define useUSCustomary directly, and useFarenheit = useUSCustomary or BS, KY, LR

(Avoiding references to international agreements and politicial entities in code can be a good idea, since those things tend to change over time.)

Reply:
I really hesitated about it, but I also don't want to assume that all countries using US Customary units also use Farenheit (this list will probably change with the times). I would prefer to rename `isPartOfCOFA` to `useUSCustomaryAndFahrenheit`.

Reply:
useUSCustomaryAndFarenheit sounds good to me.